src/css/common.css

```diff
@@ -0,0 +1,118 @@
+@import url(https://halloui.netlify.app/halloUi-share.css);
+/* @import url(/css/homepage.css);
+@import url(/css/header.css);
+@import url(/css/footer.css);
+@import url(/css/variable.css);
+@import url(/css/util.css);
+@import url(/pages/productList/productList.css);
+@import url(/pages/wishlist/wishlist.css); */
```

Comment:
Remove this as well if not using

Reply:
i just left them in case i'll need the refrence in future.